src/quo2/components/wallet/account_overview/style.cljs

```diff
@@ -0,0 +1,86 @@
+(ns quo2.components.wallet.account-overview.style)
+
+(def account-overview-wrapper
+  {:width           335
```

Comment:
:width never should be used, paddings instead

Reply:
But also let's not use margin on the outer container of a component. Better to just pass a prop 'container-style' so we can set it on the page where its being used 👍

Reply:
Can you please take another look @J-Son89